mnefun/_scoring.py

```diff
@@ -104,15 +104,28 @@ def extract_expyfun_events(fname, return_offsets=False):
     return these_events, resps, orig_events
 
 
-def _read_events(p, subj, ridx, raw):
+def _pick_events(events, picker):
+    old_cnt = sum(len(e) for e in events)
+    events = picker(events)
+    new_cnt = len(events)
+    print(f'  Using {new_cnt}/{old_cnt} events.')
```

Comment:
I'm surprised the right indentation level is not at least 4 spaces, does this print properly?

Reply:
I modified this line from previous old-style formatted string.  I think it's supposed to be slightly indented to stand apart from other logging text.  It seems to print as expected.